src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/DiagnosticDescriptorCreationAnalyzerTests.cs

```diff
@@ -162,17 +164,14 @@ class MyAnalyzer : DiagnosticAnalyzer
     private static LocalizableString dummyLocalizableTitle = new LocalizableResourceString(""dummyName"", null, null);
 
     private static readonly DiagnosticDescriptor descriptor =
-        new DiagnosticDescriptor(""MyDiagnosticId"", dummyLocalizableTitle, ""MyDiagnosticMessage"", ""MyDiagnosticCategory"", DiagnosticSeverity.Warning, isEnabledByDefault: true, helpLinkUri = ""HelpLink"");
+        new DiagnosticDescriptor(""MyDiagnosticId"", dummyLocalizableTitle, ""MyDiagnosticMessage"", ""MyDiagnosticCategory"", DiagnosticSeverity.Warning, isEnabledByDefault: true, helpLinkUri: ""HelpLink"");
```

Comment:
I am not sure whether this was a typo or a meaningful test so I went for the fix but could go for the other solution.